src/rgw/rgw_sync.cc

```diff
@@ -1084,9 +1084,10 @@ class RGWAsyncMetaStoreEntry : public RGWAsyncRadosRequest {
   rgw::sal::RGWRadosStore *store;
   string raw_key;
   bufferlist bl;
+  bool from_remote_zone;
```

Comment:
this variable looks unused - you're calling `put()` with true

Reply:
done